sdk/identity/identity/src/credentials/clientCertificateCredential.ts

```diff
@@ -65,6 +65,15 @@ export class ClientCertificateCredential implements TokenCredential {
     this.certificateX5t = Buffer.from(this.certificateThumbprint, "hex").toString("base64");
   }
 
+  private timestampInSeconds(date: Date): number {
```

Comment:
If we're moving these, looks like they could be non-exported functions rather than private methods since they don't need any class state?

Reply:
Yep, I'll change that.